03-task-manager/starter/models/Task.js

```diff
@@ -3,6 +3,16 @@ const mongoose = require('mongoose');
 const TaskSchema = new mongoose.Schema({
   name: String,
   completed: Boolean,
+  name: {
+    type: String,
+    required: [true, 'Must provide name'],
+    trim: true,
+    maxlength: [20, 'Name can not be more than 20 characters'],
+  },
+  completed: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
we don't need these lines since they are superceded by the declaration of \`name\` and \`completed\` later in this object

```suggestion
  name: {
    type: String,
    required: [true, 'Must provide name'],
    trim: true,
    maxlength: [20, 'Name can not be more than 20 characters'],
  },
  completed: {
    type: Boolean,
    default: false,
  },
```